_posts/2021-12-23-Fortran-lang-2021-in-review.md

```diff
@@ -0,0 +1,355 @@
+implemented in Haskell to a pure Fortran implementation.
+Fpm has since been used in increasingly more and ever larger Fortran projects.
+
+To date, [22 people](https://github.com/fortran-lang/fpm/graphs/contributors)
+have contributed code to fpm.
+In 2021 fpm has advanced from v0.1.3 to v0.5.0.
+Key additions this year include (release version in parentheses):
+
+* CLI arguments for linker, archiver, and C compiler (0.5.0)
+* Support for MPI and LFortran (0.4.0)
+* Support for installed library modules via `external-modules` in the manifest
+  (0.3.0)
+* Automatic discovery of manifest files in parent directories (0.3.0)
+* Support for reading package version from file (0.3.0)
+* Support for include directories (0.2.0)
+* Support for `--flag` CLI option to specify compiler flags (0.2.0)
+* `fpm build --show-model` displays the internal representation of a package
+  (0.1.4)
+* Allow hyphen in new package names (0.1.4)
+* `fpm new` now supports `--full` and `--bare` to specify level of scaffolding
+  (0.1.4)
+
+Check out also these fpm plugins:
+
+* [fpm-search](https://github.com/brocolis/fpm-search):
+  Adds the `fpm search` command for searching for registered fpm packages from
+  the command line.
+* [fpm-man](https://github.com/urbanjost/fpm-man):
+  Adds the `fpm man` command for displaying man-style help pages about Fortran
+  intrinsics and other language features.
+
+At the time of writing, there are almost 200 projects now using fpm.
+If you haven't tried fpm yet, please do!
+It has truly been a game-changing tool for many of us.
+
+## LFortran
+
+<center>
+<a href="https://lfortran.org">
+<img src="{{ site.baseurl }}/assets/img/Fortran-lang-2021-in-review/lfortran-logo-300x300.png">
+</a>
+</center>
+
+Though not technically a Fortran-lang project,
+[LFortran](https://lfortran.org) has been growing close to the Fortran-lang
+community and projects largely thanks to its creator and lead developer 
+[Ondřej Čertík](https://github.com/certik) also being one of the founding
+members of Fortran-lang.
+LFortran has been developing rapidly this year and was released as a
+[Minimum Viable Product (MVP)](https://lfortran.org/blog/2021/09/lfortran-minimum-viable-product-mvp/) in September.
+LFortran currently parses all of Fortran 2018 and compiles a significant
+subset of the language.
+If you haven't tried LFortran yet, please do and let us know how it can best
+improve for your use case.
+You can help speed up LFortran's development by contributing code and/or
+documentation directly, or by donating funds to the project via
+[NumFOCUS](https://numfocus.org/project/lfortran).
+
+Follow LFortran on Twitter at [@lfortranorg](https://twitter.com/lfortranorg).
+
+## fortran-lang.org, Discourse, and social media
+
+A major addition to the Fortran website this year is the
+[Fortran Best Practices mini-book](https://fortran-lang.org/learn/best_practices).
+This is a port and an update to the well-known
+[fortran90.org](https://fortran90.org) by
```

Comment:
fortran90.org doesn't have an up-to-date certificate, let's use http instead
```suggestion
[fortran90.org](http://fortran90.org) by
```

Reply:
The certificate seems okay:

![fortran90](https://user-images.githubusercontent.com/4133310/147517746-fd1d20fe-5145-4246-95e7-171d2d346bec.png)

Reply:
Okay, I can't connect to fortran90.org without dismissing a security warning from firefox.
```suggestion
[fortran90.org](https://www.fortran90.org) by
```
Seems that the certificate is for the www subdomain not the toplevel domain, using the link above works without warnings for me.

Reply:
I noticed the same thing, yes https://www.fortran90.org/ works, so let's use that. I can fix it later.